docs/BUILD.md

```diff
@@ -26,7 +26,7 @@ Create a PR for the new release:
 To release the library, you need to [create a new release](https://github.com/commercetools/commercetools-project-sync/releases/new) with Github, 
 describe the new release and publish it. 
 
-For example, The release description should show of new features, bug fixes and updates of the dependencies versions. check [previous version tag.](https://github.com/commercetools/commercetools-project-sync/releases/tag/3.10.0)
+For example, The release description should show of new features, bug fixes and updates of the dependencies versions. check [previous version tag.](https://github.com/commercetools/commercetools-project-sync/releases/tag/3.11.0)
```

Comment:
Replace the specific version link with the list of releases
```suggestion
The release description should show new features, bug fixes and updates of the dependencies versions. Check [previous releases](https://github.com/commercetools/commercetools-project-sync/releases).
```

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/257/commits/eda12109eae4a8d5b3e43ef308cdc75a13063964